scripts/circle-ci-tests.sh

```diff
@@ -92,6 +91,11 @@ function get_children() {
 }
 
 function run_shard_0() {
+    # run stanford-specific quality checks here
+    true
+}
+
+function run_shard_4() {
```

Comment:
Basically split everything out into stanford and non-stanford shards of each quality/lms/cms/common.

Run stanford first, so we know quickly if we broke our own stuff.